### PR TITLE
Reorganize existing content in navbar

### PR DESCRIPTION
**Pull request by logason**
Categorise all existing content into the new high level nav groups:
- Tracking Plan Audit
- Tracking Plan Management
- Tracking Observability
- Analytics Release Workflow
- Type safe code, unit tests and CI
- Reference

Content improvement opportunities [documented in this Asana project](https://app.asana.com/0/1200571264461528/1200603282068243).

### CODE REVIEW

File: components/Navigation.tsx [hunk 1]
```diff
@@ -82,22 +82,28 @@ const navigation: navigationItem[] = [
...
+      {
+        type: 'route',
+        title: 'Quickstart: Design data in Avo',
+        path: '/data-design/start-data-design',
+      },
+
       {
         type: 'group',
-        title: 'Tracking plan',
+        title: 'The Avo Tracking Plan',
```
Comment (alex-tpom6oh) [inline components/Navigation.tsx:101]: Nice to see it finally integrated in the general contents

**Comment by alex-tpom6oh** [comment]
Jul-19-2021 12-56-50

I think I'm seeing this issue with some articles not responding for the second time, if I refresh it starts to work again 🤔

**Review by alex-tpom6oh — APPROVED** [review]
Content looks great

**Comment by logason** [comment]
> I think I'm seeing this issue with some articles not responding for the second time, if I refresh it starts to work again 🤔

Oh noes I see it as well. Which browser are you in? Could be this issue still lurking around: https://app.asana.com/0/1175994043817969/1200082831476853

**Comment by hugihlynsson** [comment]
#### 📝Changed routes:


| [**/workspace**](https://docs-atlc6myby.avo-dev.app/workspace) | [**/workspace/tracking-plan**](https://docs-atlc6myby.avo-dev.app/workspace/tracking-plan) |
|:-:|:-:|
|<a href="https://docs-atlc6myby.avo-dev.app/workspace">Screenshot of /workspace</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/HylzsrvKXgTXlBDvv3f9VAYe8IhHkyp9qWwabtuGUctmoYIYBH8pWy3HasyR6PXMhHL1NpP7dc_yhf1f80VizA.png">(view full size)</a> |<a href="https://docs-atlc6myby.avo-dev.app/workspace/tracking-plan">Screenshot of /workspace/tracking-plan</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/HCbY4f3qPMUDQKtLsmnYmGqZlJ4SDEGbnAbGoab-uL_qAy56yU85A8BZvHIs_zfXYG_Nw874auOQPmmgNd5i9w.png">(view full size)</a> |





3 deleted routes:
- **/data-design/start-publishing**
- **/workspace/connections**
- **/workspace/implement**

Commit 11f61e2589af739e777dd48e8bfc8229945875e4 (https://docs-atlc6myby.avo-dev.app).

**Comment by logason** [comment]
> > I think I'm seeing this issue with some articles not responding for the second time, if I refresh it starts to work again 🤔
> 
> Oh noes I see it as well. Which browser are you in? Could be this issue still lurking around: https://app.asana.com/0/1175994043817969/1200082831476853

Looks like this is happening when we have the "Quickstart: Design data in Avo" guide open. The amount of heavy gifs being rendered at the same time are probably taking the JS engine down 😬  I'll add a task to the docs project to migrate those gifs to mp4s

**Pull request merged**